docs: new package structure for lumos (#263)
nervosnetwork/lumos: README.md

As of now, lumos contains the following components:

> - [ckb-indexer](./packages/ckb-indexer): an index base on ckb-indexer and can be used on the web side.

Reviewer: ```suggestion
- [ckb-indexer](./packages/ckb-indexer): an cell/tx indexer base on [ckb-indexer](https://github.com/nervosnetwork/ckb-indexer). 
```

- [hd](./packages/hd): an HD wallet manager for CKB. It support mnemonic and keystore, compatible with `Neuron` and `ckb-cli`, you can load keystore from `Neuron` or `ckb-cli` directly and import mnemonic generated by `Neuron`.
- [hd-cache](./packages/hd-cache): an HD cache manager for CKB. It build a memory cache for derived addresses and live cells of these addresses.
- [rpc](./packages/rpc): RPC module for CKB RPC. Provide type definations for CKB RPC interface.

examples

- [omni-lock-metamask](./examples/omni-lock-metamask/)
- [pw-lock-metamask](./examples/pw-lock-metamask/)
- [secp256k1-multisig-transfer](./examples/secp256k1-multisig-transfer/)

> - [secp256k1-transfer](./examples/secp256k1-transfer/)

Reviewer: ```suggestion
## Examples

we have provided some use cases for Lumos, such as interactions with MetaMask, transfers CKB, address conversions, etc. which you can find in [examples](./examples)
```
